server/start.go

```diff
@@ -998,6 +1000,9 @@ func addStartNodeFlags(cmd *cobra.Command, opts StartCmdOptions) {
 	cmd.Flags().Int(FlagMempoolMaxTxs, mempool.DefaultMaxTx, "Sets MaxTx value for the app-side mempool")
 	cmd.Flags().Duration(FlagShutdownGrace, 0*time.Second, "On Shutdown, duration to wait for resource clean up")
 
+	// add optimistic execution flag
+	cmd.Flags().Bool(FlagOptimisticExecutionEnabled, false, "Enable optimistic execution")
```

Comment:
do you think this is reasonable to make this flag outstanding/emphasised? why not just like the other flags?

```suggestion
	cmd.Flags().Duration(FlagShutdownGrace, 0*time.Second, "On Shutdown, duration to wait for resource clean up")
	cmd.Flags().Bool(FlagOptimisticExecutionEnabled, false, "Enable optimistic execution")
```

Reply:
Done